website/templates/home.mako

```diff
@@ -33,6 +33,7 @@
 
 <%def name="stylesheets()">
   <link rel="stylesheet" href="/static/css/pages/home-page.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1, minimum-scale=1">
```

Comment:
Prevents zooming on this page (essentially the same as an `overflow-x:hidden` style, but for mobile). There's a bit of hackery involved in getting the banner to have a wider width than the element it's contained by, making this necessary. 

We use a less restrictive version of this in base.mako: https://github.com/CenterForOpenScience/osf.io/blob/develop/website/templates/base.mako#L11.